code/core/src/telemetry/get-has-router-package.ts

```diff
@@ -0,0 +1,37 @@
+import type { PackageJson } from '../types';
+
+const routerPackages = new Set([
+  'react-router',
+  'react-router-dom',
+  'remix',
+  '@tanstack/react-router',
+  'expo-router',
+  '@reach/router',
+  'react-easy-router',
+  '@remix-run/router',
+  'wouter',
+  'wouter-preact',
+  'preact-router',
+  'vue-router',
+  'unplugin-vue-router',
+  '@angular/router',
+  '@solidjs/router',
+
+  // metaframeworks that imply routing
+  'next',
+  'react-scripts',
+  'gatsby',
+  'nuxt',
+  '@sveltejs/kit',
+]);
+
+/**
+ * @param packageJson The package JSON of the project
+ * @returns Boolean Does this project use a routing package?
+ */
+export function getHasRouterPackage(packageJson: PackageJson) {
```

Comment:
`hasRouterPackage`?

Reply:
Well the metadata field is called that, I though maybe the helper that gets it should be named differently.